tests/integration/acp/relationship/doc_actor/add/with_target_all_actors_gql_test.go

```diff
@@ -248,3 +248,209 @@ func TestACP_OwnerGivesOnlyReadAccessToAllActors_GQL_AllActorsCanReadButNotUpdat
 
 	testUtils.ExecuteTestCase(t, test)
 }
+
+func TestACP_OwnerGivesOnlyReadAccessToAllActors_GQL_CanReadEvenWithoutIdentityButNotUpdateOrDelete(t *testing.T) {
+	expectedPolicyID := "fc56b7509c20ac8ce682b3b9b4fdaad868a9c70dda6ec16720298be64f16e9a4"
+
+	test := testUtils.TestCase{
+
+		Description: "Test acp, owner gives read access to all actors (gql), can read without an identity but can't update or delete",
+
+		SupportedMutationTypes: immutable.Some(
+			[]testUtils.MutationType{
+				// GQL mutation will return no error when wrong identity is used so test that separately.
```

Comment:
suggestion: This is true for update only it seems. I suggest making the comment represent that.

Reply:
- [ ] Add that this is true for update only